milvus/datadog_checks/milvus/metrics.py

```diff
@@ -0,0 +1,226 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+METRIC_MAP = {
+    'milvus_build_info': 'build_info',
```

Comment:
Maybe this can be better used as the metadata entry? 

```
    'milvus_build_info': {'type': 'metadata', 'label': 'version', 'name': 'version'}
```

To grab the version here:

```
# HELP milvus_build_info Build information of milvus
# TYPE milvus_build_info gauge
milvus_build_info{built="Tue Nov  5 07:22:45 UTC 2024",git_commit="b7c80f9",version="v2.4.15"} 1
```

But I'm not sure which one would be better here.

Reply:
Did it as I don't understand your concern.
Better between what and what? What are we replacing by doing that?